test/evergreen.yml

```diff
@@ -2165,6 +2207,9 @@ buildvariants:
   tasks:
   - name: compile
   - name: unit-test
+  - name: format-stress-ppc-zseries-test
```

Comment:
Should the `format-stress-ppc-zseries-test` also be included under the `ubuntu1804-zseries-build` section?

Reply:
No, not in this ticket. There is seperate ticket(WT-4999) to migrate format-stress-zseries job in which I would include this task under zseries buildvariant.

Reply:
Ah okay, thanks!